snippets/kotlin/health.code-snippets

```diff
@@ -0,0 +1,100 @@
+{
+  "Initialize health": {
+    "scope": "php",
+    "prefix": "health",
+    "body": [
+      "val health = Health(client)"
+    ],
+    "description": "Create a Health object and initialize it."
+  },
+  "Get HTTP": {
+    "scope": "php",
+    "prefix": ["getHttp", "getHealth"],
+    "body": [
+      "val ${1:response} = health.get()",
+      "val ${2:json} = ${1:response}.body?.string()"
```

Comment:
I am skeptical about whether we should add this. I wanted to know does `response` variable contain other properties other than `body` ? If yes, then there is a possibility the user does not want to invoke this method at all and might want to use custom logic then.

What do you think?

Reply:
Let me look through the SDK once to check what type is being returned by the Health API.

Reply:
In this scenario, considering the response types (check  `HealthAntivirus`, `HealthStatus`, `HealthTime`, and `HealthQueue` [here](https://github.com/appwrite/sdk-for-kotlin/tree/main/src/main/kotlin/io/appwrite/models)) I believe we can let it be.

Reply:
Alright, let's keep it this way. In the  future, if these responses have more properties then we can depreciate them

Reply:
Probably something to add in the Contributing Guide so that other contributors can remember to review on a case-by-case basis?

Reply:
umm, I think it's better that this case should be reviewed in individual PRs.